Squirrely-Ducks/bangazon-sprint-3
# Bm order view

## DESCRIPTION
What did you do? 
added routes for product details from order list
Why did you do it?
to complete ticket

## STEPS TO TEST
 . Test functionality
	* log in 
	* go to my account
	* click on an order to see its products
****NOTE CURRENTLY SEQUELIZE LOGIC IS NOT CORRECT GETS ALL ORDERS EVEN IF OPEN
## REFERENCE
Does it reference a ticket?
#8
